src/Components/Notifications/NotificationsHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Box,
+  Clickable,
+  DROP_SHADOW,
+  Flex,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import CloseIcon from "@artsy/icons/CloseIcon"
+import MoreIcon from "@artsy/icons/MoreIcon"
+import { NotificationsPills } from "Components/Notifications/NotificationsPills"
+import { NotificationListMode } from "Components/Notifications/NotificationsWrapper"
+
+export interface NotificationsHeaderProps {
+  mode: NotificationListMode
+}
+
+export const NotificationsHeader: React.FC<NotificationsHeaderProps> = ({
+  mode,
+}) => {
+  return (
+    <Box
+      backgroundColor="white100"
+      width="100%"
+      p="20px 20px 0"
```

Comment:
A more elegant way to apply padding with no bottom padding?

<img width="370" alt="Screenshot 2024-01-18 at 18 37 41" src="https://github.com/artsy/force/assets/17580625/d8c4edcb-50bb-4a92-b5c2-7d33f5f3766b">

Reply:
I would rather do something like:

```
px=2
pt=2
```